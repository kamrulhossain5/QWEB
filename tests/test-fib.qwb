function int fib(int x){
  if (x < 2){
    output 1;
  }
  output fib(x-1) + fib(x-2);
}

function int main(){
  print(fib(0));
  print(fib(1));
  print(fib(2));
  print(fib(3));
  print(fib(4));
  print(fib(5));
}
