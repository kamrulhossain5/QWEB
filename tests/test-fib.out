1<br>
1<br>
2<br>
3<br>
5<br>
8<br>
